contracts/libraries/LibUtils.sol

```diff
@@ -56,14 +56,12 @@ library LibUtils {
         require(bytes(newAssetId).length > 0, "assetId must not have 0 length");
     }
 
-    function archaeologistUnwrappedCheck(
-        bytes32 identifier,
-        address archaeologist
-    ) internal view {
+    function archaeologistUnwrappedCheck(bytes32 sarcoId, address archaeologist)
```

Comment:
@DarksightKellar  --- should this stay with each argument on its own line? Elsewhere in the code multiple arguments are split onto their own lines. 

this is very minor (and not a blocker)

Reply:
@sethhrbek -- This is a line length thing, so as a line approaches 100 chars (in this setup), auto-formatting breaks it up